src/SprykerSdk/Zed/AppSdk/Communication/Console/OpenApiCreateConsole.php

```diff
@@ -76,10 +76,8 @@ public function execute(InputInterface $input, OutputInterface $output): int
         }
 
         // @codeCoverageIgnoreStart
-        if ($output->isVerbose()) {
```

Comment:
Why this got removed?

Reply:
because for showing error on console, I have to add -v at the end of openapi:create command.

Reply:
Thats expected, we only want this output when we run in verbose mode.

Reply:
okay